pasqal_cloud/__init__.py

```diff
@@ -143,12 +170,18 @@ def create_batch(
         if configuration:
             req.update({"configuration": configuration.to_dict()})  # type: ignore
 
-        batch_rsp = self._client._send_batch(req)
+        try:
+            batch_rsp = self._client._send_batch(req)
+        except HTTPError as e:
+            raise BatchCreationError(
+                f"Couldn't create batch due to a client error: {e}"
+            )
```

Comment:
Why do we sometimes pass the exception directly and sometimes pass a formatted string? Can we make it consistent?

Reply:
That's on an outdated version of the code, we do constitently pass the exception.